### PR TITLE
[framework] Remove redundant path segments (part 1)

### PR DESCRIPTION
**Pull request by nemrism**
Remove redundant path in relative links inside /framework/** files

Contributes to #10653

### CODE REVIEW

**Comment by mairaw** [comment]
I know these changes are super straightforward but let's try to keep these PRs with ~100 files. I'll review this one as-is though.

File: docs/framework/additional-apis/index.md
Comment (mairaw) [inline docs/framework/additional-apis/index.md]: I think this path needs to look like this:

```suggestion
- [The .NET Framework and Out-of-Band Releases](../get-started/the-net-framework-and-out-of-band-releases.md)
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Wonderful to see these changes. In addition to the broken link I've pointed out, I also have a few additional comments. I've reviewed 200/502 files so far.

File: docs/framework/network-programming/using-an-asynchronous-server-socket.md
Comment (mairaw) [inline docs/framework/network-programming/using-an-asynchronous-server-socket.md]:
```suggestion
- [Threading](../../standard/threading/index.md)
```

File: docs/framework/performance/etw-events.md
Comment (mairaw) [inline docs/framework/performance/etw-events.md]:
```suggestion
 In the .NET Framework, ETW event tracing is available for the common language runtime (CLR), the [Task Parallel Library](../../standard/parallel-programming/task-parallel-library-tpl.md), and [Parallel LINQ (PLINQ)](../../standard/parallel-programming/parallel-linq-plinq.md).  
```

File: docs/framework/performance/etw-events.md
Comment (mairaw) [inline docs/framework/performance/etw-events.md]:
```suggestion
- [Task Parallel Library (TPL)](../../standard/parallel-programming/task-parallel-library-tpl.md)
```

File: docs/framework/performance/lazy-initialization.md
Comment (mairaw) [inline docs/framework/performance/lazy-initialization.md]:
```suggestion
- [Managed Threading Basics](../../standard/threading/managed-threading-basics.md)
```

File: docs/framework/performance/lazy-initialization.md
Comment (mairaw) [inline docs/framework/performance/lazy-initialization.md]:
```suggestion
- [Threads and Threading](../../standard/threading/threads-and-threading.md)
```

File: docs/framework/performance/lazy-initialization.md
Comment (mairaw) [inline docs/framework/performance/lazy-initialization.md]:
```suggestion
- [Task Parallel Library (TPL)](../../standard/parallel-programming/task-parallel-library-tpl.md)
```

File: docs/framework/reflection-and-codedom/dynamic-language-runtime-overview.md
Comment (mairaw) [inline docs/framework/reflection-and-codedom/dynamic-language-runtime-overview.md]:
```suggestion
 ![Dynamic Language Runtime Architecture Overview](./media/dlr-archoverview.png "DLR_ArchOverview")
```

File: docs/framework/reflection-and-codedom/dynamically-loading-and-using-types.md
Comment (mairaw) [inline docs/framework/reflection-and-codedom/dynamically-loading-and-using-types.md]:
```suggestion
- [Type Conversion in the .NET Framework](../../standard/base-types/type-conversion.md)
```

**Review by mairaw — COMMENTED** [review]
A few more comments... 300/502 files reviewed.

File: docs/framework/tools/winres-exe-windows-forms-resource-editor.md
Comment (mairaw) [inline docs/framework/tools/winres-exe-windows-forms-resource-editor.md]:
```suggestion
- [Globalization and Localization](../../standard/globalization-localization/index.md)
```

File: docs/framework/ui-automation/access-embedded-objects-using-ui-automation.md
Comment (mairaw) [inline docs/framework/ui-automation/access-embedded-objects-using-ui-automation.md]:
```suggestion
 ![Embedded objects in a text container.](./media/uia-textpattern-embeddedobjects.PNG "UIA_TextPattern_EmbeddedObjects")  
```

**Review by mairaw — COMMENTED** [review]
More comments,,, 400/502

File: docs/framework/ui-automation/implementing-the-ui-automation-table-control-pattern.md
Comment (mairaw) [inline docs/framework/ui-automation/implementing-the-ui-automation-table-control-pattern.md]:
```suggestion
 ![Table with complex header items.](./media/uia-tablepattern-complex-column-headers.PNG "UIA_TablePattern_Complex_Column_Headers")  
```

File: docs/framework/ui-automation/implementing-the-ui-automation-table-control-pattern.md
Comment (mairaw) [inline docs/framework/ui-automation/implementing-the-ui-automation-table-control-pattern.md]:
```suggestion
 ![Table with ambiguous RowOrColumnMajor property.](./media/uia-tablepattern-roworcolumnmajorproperty.PNG "UIA_TablePattern_RowOrColumnMajorProperty")  
```

File: docs/framework/ui-automation/implementing-the-ui-automation-value-control-pattern.md
Comment (mairaw) [inline docs/framework/ui-automation/implementing-the-ui-automation-value-control-pattern.md]:
```suggestion
 ![Editable list item.](./media/uia-valuepattern-editable-listitem.PNG "UIA_ValuePattern_Editable_ListItem")  
```

File: docs/framework/ui-automation/implementing-the-ui-automation-value-control-pattern.md
Comment (mairaw) [inline docs/framework/ui-automation/implementing-the-ui-automation-value-control-pattern.md]:
```suggestion
 ![Color picker with yellow highlighted.](./media/uia-valuepattern-colorpicker.png "UIA_ValuePattern_ColorPicker")  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
 ![Embedded Table with Image in a Text Container](./media/uia-textpattern-embedded-objects-overview-example1.png "UIA_TextPattern_Embedded_Objects_Overview_Example1")  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
 ![Content view for the preceding example](./media/uia-textpattern-embedded-objects-overview-example2.PNG "UIA_TextPattern_Embedded_Objects_Overview_Example2")  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
 ![Text ranges spanned by embedded objects.](./media/uia-textpattern-embeddedobjecttextranges.png "UIA_TextPattern_EmbeddedObjectTextRanges")  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
 ![Range adjustments by Move & ExpandToEnclosingUnit](./media/uia-textpattern-moveandexpand-examples.png "UIA_TextPattern_MoveAndExpand_Examples")  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
 {The image ![Embedded Image Example](./media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample") is embedded in text}.  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
 {The image} ![Embedded Image Example](./media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample") is embedded in text.  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
|![Embedded Image Example](./media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample")|X|  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
|![Embedded Image Example 2](./media/uia-textpattern-embedded-objects-overview-imageexample2.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample2")|Y|  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
|![Embedded Image Example 3](./media/uia-textpattern-embedded-objects-overview-imageexample3.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample3")<br /><br /> Image for Z|Z|  
```

File: docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md
Comment (mairaw) [inline docs/framework/ui-automation/textpattern-and-embedded-objects-overview.md]:
```suggestion
|<xref:System.Windows.Automation.TextPattern.RangeFromChild%2A> where <xref:System.Windows.Automation.AutomationElement> is the object returned by the previous `GetItem` method.|Returns the range that spans the image ![Embedded Image Example](./media/uia-textpattern-embedded-objects-overview-imageexample.PNG "UIA_TextPattern_Embedded_Objects_Overview_ImageExample").|  
```

File: docs/framework/ui-automation/ui-automation-support-for-the-pane-control-type.md
Comment (mairaw) [inline docs/framework/ui-automation/ui-automation-support-for-the-pane-control-type.md]:
```suggestion
 ![Screenshot of applet window with two panes](./media/uiauto-pane.GIF "uiauto_pane")  
```

File: docs/framework/ui-automation/ui-automation-support-for-the-splitbutton-control-type.md
Comment (mairaw) [inline docs/framework/ui-automation/ui-automation-support-for-the-splitbutton-control-type.md]:
```suggestion
 ![Split button](./media/uiauto-splitbutton-detailed.gif "uiauto_splitbutton_detailed")  
```

File: docs/framework/ui-automation/ui-automation-textpattern-overview.md
Comment (mairaw) [inline docs/framework/ui-automation/ui-automation-textpattern-overview.md]:
```suggestion
![TextPatternRangeEndpoints &#40;start and end&#41;.](./media/uia-textpattern-endpoints.PNG "UIA_TextPattern_Endpoints")
```

File: docs/framework/windows-services/how-to-create-windows-services.md
Comment (mairaw) [inline docs/framework/windows-services/how-to-create-windows-services.md]:
```suggestion
     ![Set the ServiceName property.](./media/windowsservice-servicename.PNG "WindowsService_ServiceName")  
```

**Review by mairaw — COMMENTED** [review]
Finished the review now. A couple more comments...

Also, apologies for including two commits here that were supposed to be done on a separate PR I'm working on. It's fine to leave them here if they don't cause any issues.

**Comment by nemrism** [comment]
I applied all suggested fixes about "incorrect" includes of `media` files.

I've changed all links to `/standard` section in the following PR: #13925 as this replacement work is quite huge even if I split by folders...
Once it will be merged I could rebase this branch in order to "facilitate" the final review.

**Comment by mairaw** [comment]
@nemrism can you fix the merge conflicts?

**Comment by nemrism** [comment]
Yes, I'll push that asap

**Comment by nemrism** [comment]
Merged, resolved and ready for your review @mairaw

**Comment by mairaw** [comment]
Thanks @nemrism. @rpetrusha I'm afraid this will cause new merge conflicts for #13851. Should we hold off until that one is merged?

**Comment by nemrism** [comment]
cc @mairaw / @rpetrusha 

Just tested the [v-thepet/move2](https://github.com/v-thepet/docs/tree/move2) branch against this one. It is indeed the most impacted one by merge conflicts but it is quite easy to resolve them.

Let me know when the #13851 PR will be merged, I'll handle the conflicts.

I'll continue to work on the #10653 resolution, it seems that other section are less impacted, automatic merge works fine on most of other related PR.

**Comment by mairaw** [comment]
That PR was now merged. Can you please rebase your PR? Thanks!

**Comment by nemrism** [comment]
Yes of course!
I'll prepare that and tell you when this one is ready for final review.

**Comment by nemrism** [comment]
All done, ready for final review @mairaw

**Review by mairaw — APPROVED** [review]
Changes look good! Thank you @nemrism!

**Pull request merged**
